### DISCUSSION TITLE
Saleor x Next.js Commerce

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#7463] [Announcements]
The future of commerce is here and it's fast! Check out how Saleor can work together with Vercel's Nextjs Commerce.
The official provider is here along with the built-in integration!

https://saleor-commerce.vercel.app/

--------------------
- [Twitter](https://twitter.com/getsaleor/status/1402977586892918784)
- [LinkedIn](https://www.linkedin.com/feed/update/urn:li:activity:6808745689237987328)

**Comment by karolkielecki** [comment]
PS checkout is not ready yet, work-in-progress. It will be available soon!

**Reply by heliumbrain** [reply]
Nice work! Can't wait until the checkout is implemented as well :)

Gonna give it a spin. 

Btw, I noticed that the link to the demo on https://github.com/vercel/commerce point to https://saleor.vercel.app/ which doesn't exist.

**Reply by karolkielecki** [reply]
@heliumbrain Thanks for your kind words and reporting! I've passed the issue to our team--thanks! Cheers :)

**Comment by janfeldmann** [comment]
Hey, sounds really good. I found out about Nextjs Commerce and Saleor a couple of weeks ago and really like the combination.
I am currently testing a checkout implementation with Mollie in this fork feature:
https://github.com/janfeldmann/nextjs-commerce/tree/saleor-checkout/framework/saleor
So far I am able to create a payment and an order from the Nextjs Commerce frontend that is then send to Saleor and Mollie.
It is still work in progress with a lot of static data for testing things out.

**Reply by muhammadarslan** [reply]
https://saleor.vercel.store/ i can see this has implemented the checkout does this work with 3.1?
